addons/l10n_lv/data/account.account.template.csv

```diff
@@ -0,0 +1,263 @@
+"id","name","code","account_type","chart_template_id/id","tag_ids/id","reconcile"
+"a1110","Pētniecības un uzņēmuma attīstības izmaksas",1110,"asset_non_current","l10n_lv.account_chart_template_latvia","","False"

```

Comment:
If possible, It is preferred to have english names for accounts and taxes (that are translated) as it makes it easier to maintain.